charts/osm/values.schema.json

```diff
@@ -397,6 +397,15 @@
                     "examples": [
                         true
                     ]
+                },
+                "osmNamespace": {
+                    "$id": "#/properties/OpenServiceMesh/properties/osmNamespace",
+                    "type": "string",
+                    "title": "The namespace of the osm controller",
+                    "description": "Indicates the namespace in which the osm controller will be installed.",
```

Comment:
nit:
title: "The namespace of the osm control plane components"
description: "Indicates the namespace in which the osm control plane components will be installed

Reply:
I'll update this nit separately so I don't dismiss stale reviews.